src/Akka.HealthCheck.Persistence/AkkaPersistenceLivenessProbe.cs

```diff
@@ -166,19 +177,22 @@ protected override bool Receive(object message)
 
         protected override void PreStart()
         {
-            CreateProbe(true);
+            CreateProbe();
         }
 
-        private void CreateProbe(bool firstTime)
+        private void CreateProbe()
         {
-            _probe = Context.ActorOf(Props.Create(() => new SuicideProbe(Self, firstTime, _id)));
-            if(firstTime)
+            var isEven = _probeCounter % 2 == 0;
```

Comment:
Need to add comments explaining the process here. Not obvious just from reading the code.